spec/services/markdown_converter_spec.rb

```diff
@@ -126,5 +126,22 @@
         expect(described_class.new(markdown).as_html).to eq(html_result)
       end
     end
+
+    context 'when the markdown includes backtick fenced codeblocks' do
+      it 'converts the markdown with appropriate code format' do
+        markdown = <<~MARKDOWN
+          ```ruby
+          puts 'this should work'
+          ```
+        MARKDOWN
+
+        html_result = <<~HTML
+          <div class="language-ruby highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="nb">puts</span> <span class="s1">'this should work'</span>
+          </code></pre></div></div>
```

Comment:
This might look kind of strange if you're familiar with what actually gets rendered to the page at the end. But what gets rendered in the end is extensively decorated by the Prisma.js syntax highlighter library on the frontend. This is the correct format for what Kramdown parses out.